app/controllers/package.scala

```diff
@@ -14,28 +14,34 @@
  * limitations under the License.
  */
 
-import cats.data.EitherT
 import controllers.actions.AuthorisedAction
 import models.domain.ApiResultT
 import models.error.ServiceError
 import play.api.Logger
 import play.api.libs.json.Format.GenericFormat
-import play.api.libs.json.{JsError, JsResult, JsSuccess, Json, Reads}
-import play.api.mvc.Results.{BadRequest, InternalServerError}
+import play.api.libs.json._
+import play.api.mvc.Results._
 import play.api.mvc.{AnyContent, Result}
 
 import scala.concurrent.{ExecutionContext, Future}
 
 package object controllers {
```

Comment:
Should we add UTs for package objects?

Reply:
yes once it stabilizes.

right now we will test it by using it in controllers - writing test controller will test some of the helpers